appveyor.yml

```diff
@@ -1,19 +1,33 @@
 version: 1.1.{build}
 os: Visual Studio 2015
+configuration: Release
+pull_requests:
+  do_not_increment_build_number: true
+
+environment:
+  global:
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true

```

Comment:
You only need to do this when first installing `dotnet`, as it's already on the box, shouldn't make any diff.